code/modules/food_and_drinks/kitchen_machinery/food_cart.dm

```diff
@@ -34,17 +34,21 @@
 	dat += "Remaining glasses: [glasses]<br>"
 	dat += "Portion: <a href='?src=[REF(src)];portion=1'>[portion]</a><br>"
 	for(var/datum/reagent/R in reagents.reagent_list)
+	for(var/i=1 to length(reagents.reagent_list))
```

Comment:
Did you mean to remove `for(var/datum/reagent/R in reagents.reagent_list)` ?
Also, use `i in 1 to ...` instead of `i=1 to...` so we don't proliferate multiple standards.

Reply:
I hate byond code.